examples/k3s.yaml

```diff
@@ -41,7 +41,15 @@ provision:
   script: |
     #!/bin/sh
     curl -sfL https://get.k3s.io | sh -
-
+- mode: system
+  script: |
+    #!/bin/bash
+    # Increase inotify limits to prevent nested Kubernetes control planes
+    # from crash looping.
```

Comment:
Is this needed for k3s? If so, it should be needed for `k8s.yaml` too?

Reply:
As far as I know, it is only needed for k3d and kind - not for k3s and k8s